catroweb.yaml

```diff
@@ -1427,6 +1455,26 @@ components:
           type: string
           example: 'https://share.catrob.at/resources/featured/featured_999.png'
 
+    ProjectsCategoryResponse:
+      type: array
+      items:
+        $ref: '#/components/schemas/ProjectsCategory'
+
+    ProjectsCategory:
+      type: object
+      properties:
+        id:
```

Comment:
the `id` can be removed here, the `name` is already a unique identifier.

Reply:
The id will be the name of the category in English. the name we return shall be the translation to the given language.

Reply:
Then I would rather choose names that give a clear indication of this.
id is name, and name is translated category title sounds a bit unintuitive to me.
At least it should be explained in the specification